docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+    "hm-admin":
+        build: ./docker/spa
```

Comment:
As [this Dockerfile is empty](https://github.com/coopTilleuls/hm-admin/pull/20/files#diff-6cb92acbc63641f885b8127e6baebec3R1), there is no need to use `build`. You could directly use `image`:
```yml
services:
    hm-admin:
        image: node:6
```